main.cpp

```diff
@@ -116,10 +96,33 @@ struct FloatType
         pointerToFloatValue = nullptr;
     }
    
-    FloatType& add ( float );
-    FloatType& subtract ( float );
-    FloatType& multiply ( float );
-    FloatType& divide ( float );
+    FloatType& operator +=(  const float x )
+    {
+        *pointerToFloatValue += x;
+        return  *this;
+    }
+
+    FloatType& operator -=(  const float x )
```

Comment:
this looks like a text emoji.  
- [ ] add a space between your operator and your `(`